services/apps/alcs/src/providers/email/email.service.ts

```diff
@@ -182,7 +183,7 @@ export class EmailService {
         errorMessage = e.response.data.detail;
       }
 
-      this.repository.save(
+      await this.repository.save(
```

Comment:
Is it critical that data will not be recorded into db if save fails in a catch?

Reply:
I dunno, I can remove this